### PR TITLE
crowbar: Update for Crowbar::Settings.domain new API

### PR DESCRIPTION
**Pull request by vuntz**
ChefObject.cloud_domain got moved to Crowbar::Settings.domain.

Needed because https://github.com/crowbar/crowbar-core/pull/519 was merged.

### CODE REVIEW

File: crowbar_framework/app/models/rabbitmq_service.rb [hunk 1]
```diff
@@ -90,7 +90,7 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
         raise "Internal error: HA enabled, but element is not a cluster"
       end
       cluster = rabbitmq_elements[0]
-      rabbitmq_vhostname = "#{role.name.gsub("-config", "")}-#{PacemakerServiceObject.cluster_name(cluster)}.#{ChefObject.cloud_domain}".gsub("_", "-")
+      rabbitmq_vhostname = "#{role.name.gsub("-config", "")}-#{PacemakerServiceObject.cluster_name(cluster)}.#{Crowbar::Settings.domain}".gsub("_", "-")
```
Comment (vuntz) [inline crowbar_framework/app/models/rabbitmq_service.rb]: Hrm, no, can't use `tr` here...

Comment (tboerger) [inline crowbar_framework/app/models/rabbitmq_service.rb]: Yes you  can

Comment (vuntz) [inline crowbar_framework/app/models/rabbitmq_service.rb]: I'm happy to learn something new if you tell me how:
```
irb
irb(main):001:0> "nova-config-default".tr("-config", "")
=> "vadeault"
```

Comment (vuntz) [inline crowbar_framework/app/models/rabbitmq_service.rb]: Doh, didn't see the other `gsub` at the end... I blame the long line :-)

**Comment by nicolasbock** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Pull request merged**